content/docs/setup/kubernetes/install/operator/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Operator Install 
+description: Install and configure Istio using the Istio Operator.
+weight: 25
+keywords: [operator,kubernetes,helm]
+---
+
+Follow this guide to install and configure an Istio mesh using an alternate
+installation method: the Istio {{<gloss operator>}}Operator{{</gloss>}}
+installation.
+
+The Istio Operator installation process has several benefits:
+
+- Common install tasks require fewer steps, some are a simple one-line command.
+
+- Strong user input validation to help prevent installation errors.
+
+- Rich customization options to easily override any aspect of the configuration.
+
+- User-friendly error messages to spot configuration errors easier.
+
+- Smart installer capable of waiting for Istio pods and services to be in a
+  ready state.
+
+- Fewer dependencies (doesn’t require Helm).
+
+The Operator install is accessed via [istioctl](/docs/ops/setup/istioctl/)
+commands and a new Operator API.
+
+{{< tip >}}
+The Istio Operator is a replacement for Helm templates and Tiller
+installation/operation.
+{{< /tip >}}
+
+## Prerequisites
+
+Before you begin, you must meet the following prerequisites.
+
+1. [Download the Istio release](/docs/setup/kubernetes/#downloading-the-release).
+1. Perform any necessary [platform-specific setup](/docs/setup/kubernetes/platform-setup/).
+1. Check the [Requirements for Pods and
+   Services](/docs/setup/kubernetes/additional-setup/requirements/).
+
+## Install Istio in One Line
+
+The simplest option is to install Istio using a one-line command.
+
+At the command prompt, run the following command:
+
+`istioctl manifest apply`
+
+When you use this option, Istio performs the following actions:
+
+- Applies the following built-in default manifests for the following components:
+    - `Injector`
+    - `Telemetry`
+    - `Citadel`
+    - `Policy`
+    - `IngressGateway`
+    - `EgressGateway`
+    - `Galley`
+    - `CertManager`
+    - `Pilot`
+    - `NodeAgent`
+    - `crds`
+
+- Installs the default profile into the cluster.
```

Comment:
Is this the installing the "default" profile or the "demo" profile? How is the "default profile" related to the "default manifests" in the previous bullet? It seems like these should not be 2 separate bullets.

Reply:
addressed